G-Chilie/PropertyPro-lite
# #166810221 Build out Page where User(agent) can Mark Posted Advert as Sold or Delete an Advert.

#### What does this PR do?

This PR is a page where the user agent can mark posted advert as sold or delete an advert.

#### Description of Task to be completed?

An icon that depicts delete and mark as sold

#### How should this be manually tested?

When you click on the icons that depict delete or the icon that depicts sold, 
it should delete or mark the items respectively

#### What are the relevant pivotal tracker stories?

[#166810221](https://www.pivotaltracker.com/story/show/166810221)

#### Screenshots (if appropriate)

![Screenshot from 2019-06-20 19-23-00](https://user-images.githubusercontent.com/39604096/59872789-6f1a1780-9392-11e9-9102-a6023a8124cb.png)
